core/templates/dev/head/components/state-directives/response-header/response-header.directive.html

```diff
@@ -69,21 +69,21 @@
 
   <br class="break-in-mobile">
   <span ng-if="$ctrl.getOutcome()">→</span>
-  <span ng-if="$ctrl.isCorrect() && $ctrl.isInQuestionMode()">
+  <span ng-if="$ctrl.isCorrect() && $ctrl.StateEditorService.isInQuestionMode()">
```

Comment:
I don't think we should be referencing services directly in the templates. Can you alias ctrl.isInQuestionMode to this function and keep the code as it previously was?

Reply:
Done